ggshield/core/scan/commit_utils.py

```diff
@@ -235,3 +235,37 @@ def parse_patch(
             else:
                 msg = f"Could not parse patch: {exc}"
             raise PatchParseError(msg)
+
+
+def get_file_sha_in_ref(
+    ref: str,
+    files: List[str],
+    cwd: Optional[Path] = None,
+) -> Iterable[Tuple[str, str]]:
+    """
+    Helper function to get the shas of files in the git reference.
+    """
+    ls_tree_output = git(["ls-tree", "-z", ref] + files, cwd=cwd)
```

Comment:
super-nit: the nice thing about helper functions is you can use shorter variable names :), here `ls_tree_output` can be `output` without ambiguity.

Reply:
You are right, so many advantages to helper functions :sparkles: